app/models/buffer_update.rb

```diff
@@ -54,6 +65,10 @@ def self.twitter_default_text(article)
     ].compact.join.strip
   end
 
+  def self.wich_social_service(service_name)
+    SOCIAL_SERVICE.fetch(service_name)
+  end
```

Comment:
side from the typo, I don't think we need the method at all, you can simply call `SOCIAL_SERVICES[social_service_name]` in the `buff!` method